plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/explorer/OpenShiftExplorerContentProvider.java

```diff
@@ -181,13 +171,7 @@ public Object getParent(Object element) {
 
 	@Override
 	public boolean hasChildren(Object element) {
-		return element instanceof ConnectionsRegistry;
-//		return false;
-//		return element instanceof IConnection
-//				|| element instanceof Project
-//				|| element instanceof ResourceGrouping
-//				|| element instanceof IDomain
-//				|| element instanceof IApplication;
+		return element instanceof IConnection;
```

Comment:
Shouldnt we check the size of the children in connection or are there always child items on the backend?